src/deploy/osp_deployer/settings_sanity.py

```diff
@@ -50,7 +51,7 @@ def check_files(self):
         logger.debug("Check settings ip's are valid.")
         shouldbbevalidips = [
             'public_api_gateway',
-            'public_api_netmask', 'public_api_allocation_pool_start',
+            'public_api_netmask',
```

Comment:
We still want public_api_allocation_pool_start in here don't we?

Reply:
That particular validation moved into the profiles json file so no longer required here.